app/scripts/views/mixins/signin-mixin.js

```diff
@@ -64,6 +67,21 @@ define(function (require, exports, module) {
           }
 
           return self.onSignInSuccess(account);
+        })
+        .fail((err) => {
+          function isUnblockableError(err) {
+            return err.verificationMethod === 'email-captcha';
```

Comment:
This should go into [lib/verification-methods.js](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/verification-methods.js).